docs-2.0/3.ngql-guide/15.text-based-index-statements/4.search-with-fulltext-index.md

```diff
@@ -0,0 +1,59 @@
+# Full-text search
+
+```ngql
+LOOKUP ON {<tag> | <edge_type>} WHERE <expression> [YIELD <return_list>]
+
+<expression>
+    PREFIX | WILDCARD | REGEXP |FUZZY
+
+<return_list>
+    <col_name> [AS <col_alias>] [, <col_name> [AS <col_alias>] ...]
+```
+
+Use the `LOOKUP ON` statement to do full-text search. The search string is specified in the `WHERE` clause.
+
+## Natural language full-text search
+
+A natural language search interprets the search string as a phrase in natural human language. The search is case-insensitive.
+
+## Examples
+
+```ngql
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a");
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a");
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a", 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 1);
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE WILDCARD(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE REGEXP(t1.c1, "a", 1, 2);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 1, 2);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", AUTO, AND);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", AUTO, OR);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, AND);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR);
+
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, -1, 1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 0, OR, 1, -1);
+nebula> LOOKUP ON t1 WHERE FUZZY(t1.c1, "a", 4, OR, 1, 1);
+
+nebula> LOOKUP ON t1 WHERE PREFIX(t1.c1, "a", AUTO, AND);
+```
+
+## Full-text search limitations
```

Comment:
PREFIX(schema_name.column_name, prefix_string, row_limit, timeout)
WILDCARD(schema_name.column_name, wildcard_string, row_limit, timeout)
REGEXP(schema_name.column_name, regexp_string, row_limit, timeout)
FUZZY(schema_name.column_name, fuzzy_string, fuzziness, operator,  row_limit, timeout)
fuzziness ：optional, default is AUTO. 
operator: optional, default is OR.
- Common parameter：
1. schema_name.column_name
2. row_limit : optional , default is 100
3. timeout : optional, default is 200ms.